py-polars/tests/unit/io/test_parquet.py

```diff
@@ -43,10 +43,12 @@ def test_round_trip(df: pl.DataFrame) -> None:
 
 
 @pytest.mark.write_disk()
-def test_write_parquet_using_pyarrow_9753(tmpdir: Path) -> None:
+def test_write_parquet_using_pyarrow_9753(tmp_path: Path) -> None:
+    tmp_path.mkdir(exist_ok=True)
```

Comment:
Is this really needed?

Reply:
It shouldn't be, but I ran into some issues before when I left it out with spurious failures. Maybe that's no longer an issue though. I'll try removing all those mkdir statements in a separate PR and see if anything breaks.